site2/docs/developing-binary-protocol.md

```diff
@@ -63,6 +63,14 @@ It is created by the broker when the message arrived at the broker and passed wi
 | `broker_timestamp` | Optional        | The timestamp when a message arrived at the broker (`id est` as the number of milliseconds since January 1st, 1970 in UTC)      |
 | `index`            | Optional        | The index of the message. It is assigned by the broker.
 
+If you want use broker entry metadata for **brokers**, configure the [`brokerEntryMetadataInterceptors`](reference-configuration.md#broker) parameter in the `broker.conf` file.
```

Comment:
want to use

Reply:
@Shoothzj 
oh, sorry my bad, a careless mistake. have updated. PTAL